multi-agent/adapters/outbound/channels/local/input_channel.py

```
@@ -0,0 +1,92 @@
+    def session_id(self) -> str:
+        return self._session_id
+
+    def emit(self, data: Any) -> None:
+        if self._closed:
+            return
+        self._queue.put(("data", data))
+
+    def is_active(self) -> bool:
+        return not self._closed
+
+    def close(self, *, cancelled: bool = False) -> None:
```

Ruff (0.15.20)

[warning] Unused method argument: `cancelled`

(ARG002)